comptests/prov/test_prov_k8s_cortx.py

```diff
@@ -352,6 +355,7 @@ def test_32940(self):
             LOGGER.info("Cluster not in good state, trying to restart it.")
             resp = self.ha_obj.cortx_start_cluster(self.master_node_list[0],
                                                    dir_path=self.prov_deploy_cfg["git_remote_path"])
+            time.sleep(100)
```

Comment:
why don't u poll instead? we have poll_cluster_status which can be used instead of flat sleep time.

Reply:
We can't use poll as it is using /root/deploy-scripts/k8_cortx_cloud/ path, but our path is /root/cortx-k8s/k8_cortx_cloud